pkg/service/repair/progress.go

```diff
@@ -55,14 +48,14 @@ type dbProgressManager struct {
 	total          atomic.Float64   // Total weighted repair progress
 	tableSize      map[string]int64 // Maps table to its size
 	totalTableSize int64            // Sum over tableSize
-	keyspaceRanges map[string]int64 // Maps keyspace to its range count
+	tableRanges    map[string]int   // Maps keyspace to its range count
```

Comment:
Is the comment still valid ? If it is, then why `tableRanges` ?

Reply:
It should be: `Maps table to its range count`.